shell/lib/db-deprecated.js

```diff
@@ -74,6 +74,7 @@ if (Meteor.isServer) {
 if (Meteor.isClient) {
   Session.setDefault("shrink-navbar", false);
   globalGrains = new ReactiveVar([]);
+  Session.set("shrink-navbar", window.localStorage.getItem("shrink-navbar") === "true");
```

Comment:
No, and I guess it's theoretically possible to listen to `storage` events across your other tabs, but I'd say that's outside the scope of this quick fix. Another solution is to persist it into mongo as a user preference and get reactivity by means of that, but the discussion on #1525 seemed to decide that localStorage was best.